pkgs/development/tools/literate-programming/Literate/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchgit, dmd, dub }:
+
+stdenv.mkDerivation {
+  name = "Literate-2017-05-28";
+
+  src = fetchgit {
+    url = "https://github.com/zyedidia/Literate.git";
+    rev = "23928d64bb19b5101dbcc794da6119beaf59f679";
+    sha256 = "094lramvacarzj8443ns18zyv7dxnivwi7kdk5xi5r2z4gx338iq";
+  };
+
+  buildInputs = [ dmd dub ];
+
+  installPhase = "install -D bin/lit $out/bin/lit";
+
+  meta = with stdenv.lib; {
+    description = "A literate programming tool for any language";
+    homepage    = http://literate.zbyedidia.webfactional.com/;
+    license = licenses.mit;
+    platforms = platforms.unix;
```

Comment:
I adapted your package a little to fit our coding standards. Do you want to maintain it?

Reply:
Thank you. I appreciate the corrections. I notice the CI is still failing. I'm not sure of the best way to proceed regarding that. The specific failure I'm not familiar with. Do you have any suggestions?

Reply:
ignore the CI, for the moment.